Cargo.lock

```diff
@@ -511,24 +524,25 @@ version = "0.11.1"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "b1d1a86f49236c215f271d40892d5fc950490551400b02ef360692c29815c714"
 dependencies = [
- "generic-array 0.14.6",
+ "generic-array 0.14.7",
  "subtle",
 ]
 
 [[package]]
 name = "cumulus-primitives-core"
 version = "0.1.0"
-source = "git+https://github.com/paritytech/cumulus?branch=polkadot-v0.9.39#d6eef144421ef5c3f339f681484d06bb729dfa82"
+source = "git+https://github.com/paritytech/cumulus?branch=polkadot-v0.9.42#f603a61ff370fc33740c9373833c3c6ba1486846"
 dependencies = [
  "parity-scale-codec",
  "polkadot-core-primitives",
  "polkadot-parachain",
  "polkadot-primitives",
+ "scale-info",
  "sp-api",
  "sp-runtime",
- "sp-std 5.0.0",
+ "sp-std 5.0.0 (git+https://github.com/paritytech/substrate?branch=polkadot-v0.9.42)",
  "sp-trie",
- "xcm",
+ "xcm 0.9.42",
```

Comment:
This should be investigated, we seem to have divergent substrate versions. Hint: Always check that the added and deleted lines are somewhat net-zero.

Reply:
Thanks Chris for the tip, I believe I found the culprit:
`xcm` related things have `release-vX.Y.Z` branch names instead of `polkadot-vX.Y.Z`, which is why they were left untouched, resulting in duplicates.

Reply:
haha, happened to me too. :)